neopixel.py

```diff
@@ -36,6 +36,12 @@
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_NeoPixel.git"
 
+# Pixel color order constants
+RGB = (0, 1, 2)
+GRB = (1, 0, 2)
+RGBW = (0, 1, 2, 3)
+GRBW = (1, 0, 2, 3)
+
 class NeoPixel:
     """
     A sequence of neopixels.
```

Comment:
The parameters here should be updated to reflect the new addition.

Reply:
Yep. I'll update the docs if this flies. At this point I'm wondering if this general approach is even good.

Reply:
Didn't even notice this PR was open. I just put in a PR to update the docs to the new format. Suggest waiting until that is merged to update doc info.